documentation/source/user_section/tutorials/3/3-1.rst

```diff
@@ -0,0 +1,22 @@
+CSA (Averaged across vertebral levels)
+######################################
+
+First, we compute the cord cross-sectional area (CSA) and average it between C3 and C4 vertebral levels.
```

Comment:
I know it is explained a bit more elsewhere in the RTD, but maybe a quick definition of a vertebral level? Also, why C3 and C4, and not averaged over all vertebral levels? Or over C1 and C2, for example?

Edit: I reread the command and realized that we are doing C3 and C4 just as an example. 😅

Reply:
> Edit: I reread the command and realized that we are doing C3 and C4 just as an example. sweat_smile

This could definitely use another sentence to make it clearer on first read. :slightly_smiling_face:

Reply:
Addressed here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3094/commits/b951cfacf8343be5742d1da68117c717ef6568e8 in `documentation/source/user_section/tutorials/shape-metric-computation/csa-averaged.rst`

Looks good @joshuacwnewton !